sycl/test-e2e/Basic/built-ins/nextafter.cpp

```diff
@@ -0,0 +1,88 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/sycl.hpp>
```

Comment:
To save a bit of time on compilation, E2E tests are not allowed to include `sycl/sycl.hpp`. It should instead include `sycl/detail/core.hpp` and whatever headers are needed. I suspect you'll need at least a builtins header.

Reply:
fixed